packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -507,13 +507,10 @@ class IOSDeviceLogReader extends DeviceLogReader {
     if (device.majorSdkVersion < _minimumUniversalLoggingSdkVersion) {
       return;
     }
-    // The VM service will not publish logging events unless the debug stream is being listened to.
-    // onDebugEvent listens to this stream as a side effect.
-    unawaited(connectedVmService.onDebugEvent);
-    _loggingSubscriptions.add((await connectedVmService.onStdoutEvent).listen((ServiceEvent event) {
-      final String logMessage = event.message;
-      if (logMessage.isNotEmpty) {
-        _linesController.add(logMessage);
+    _loggingSubscriptions.add(connectedVmService.onStdoutEvent.listen((vm_service.Event event) {
```

Comment:
This should be tested manually

Reply:
Hmm. I don't quite follow. What has to be tested manually?

Reply:
This is tested now